mosip-vid-policy.json

```diff
@@ -23,7 +23,7 @@
             "vidType": "OneTimeUse",
             "vidPolicy": {
                 "validForInMinutes": null,
-                "transactionsAllowed": 1,
+                "transactionsAllowed": 2,

```

Comment:
why are we doing this change ?

Reply:
This is to fix automation failure for generating vid for one time use vid. Actually, I have observed if the transaction allowed is 1 then it's allowing 0 times means no transaction is allowed and if transaction is 2 then only 1 transaction is allowed. so currently if transaction allowed is 1 then it will not allow to generate vid causing automation failure.

Reply:
Then its a bug @kameshsr do we have a ticket for this issue?

Reply:
https://mosip.atlassian.net/browse/MOSIP-35270